src/acl.c

```diff
@@ -2572,11 +2572,11 @@ int ACLSaveToFile(const char *filename) {
 void ACLLoadUsersAtStartup(void) {
     if (server.acl_filename[0] != '\0' && listLength(UsersToLoad) != 0) {
         serverLog(LL_WARNING,
-            "Configuring Redis with users defined in redis.conf and at "
+            "Configuring %s with users defined in valkey.conf and at "
```

Comment:
It's a little bit strange that with the macro, it's possible for a user to patch the macro, like `#define SERVER_TITLE "Redis"`, to make this message say "Redis" but it will still contain "valkey.conf" so the message is still not identical to in Redis. But we decided that we don't need to be 100% compatible in these log messages, I'm going to ignore that. We need to finish this rebranding some time. :)